Modules/Quotas/test/Quotas.Application.Tests/Tests/Quotas/DeleteTierQuotaDefinition/HandlerTests.cs

```diff
@@ -47,14 +43,18 @@ public async Task Triggers_TierQuotaDefinitionDeletedDomainEvent()
         await handler.Handle(command, CancellationToken.None);
 
         // Assert
-        A.CallTo(() => _eventBus.Publish(A<DomainEvent>.That.IsInstanceOf(typeof(TierQuotaDefinitionDeletedDomainEvent)))).MustHaveHappenedOnceExactly();
+        tier.DomainEvents.Should().HaveCount(2);
+        tier.DomainEvents[1].Should().BeOfType<TierQuotaDefinitionDeletedDomainEvent>();
+        var domainEvent = (TierQuotaDefinitionDeletedDomainEvent)tier.DomainEvents[1]; //TODO: Timo (Should I make this a domain test as well or is this handler test enough?)
```

Comment:
> Should I make this a domain test as well or is this handler test enough?

As I already said in the other PR: the domain events tests in general should ALWAYS be ONLY domain tests (obviously, as they are domain logic)